### PR TITLE
bug fix in evolve

### PR DESCRIPTION
**Pull request by niicojs**
### Short Description:

bug fix
it crashed every time
## 
## 
- 

@OpenPoGo/maintainers

### CODE REVIEW

File: plugins/evolver/__init__.py [hunk 1]
```diff
@@ -34,17 +34,18 @@ def _do_evolve(bot, name):
...
                 response = bot.api_wrapper.call()
                 if response['evolution'].success:
-                    del pokemon_evolve[0]
                     pokemon_candies -= (num_evolve - 1)
                     num_evolved += 1
                     evolved_id = response['evolution'].get_pokemon().pokemon_id
                     _log('Evolved {} into {}'.format(base_name, bot.pokemon_list[evolved_id - 1]['Name']))
 
-                    manager.fire_with_context('pokemon_evolved', bot, pokemon=pokemon_evolve[0])
+                    manager.fire_with_context('pokemon_evolved', bot, pokemon=pokemon_evolve[0], evolution=evolved_id)
```
Comment (wchill) [inline plugins/evolver/__init__.py]: `pokemon_evolve[0]` should be `pokemon`

**Comment by niicojs** [comment]
true, worst bug fix ever

**Comment by MaxLeiter** [comment]
:+1: LGTM
Please rebase

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/241/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by wchill** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/241/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Pull request merged**